Sources/HTTPConnection.swift

```diff
@@ -262,7 +262,7 @@ public func == (lhs: HTTPConnection, rhs: HTTPConnection) -> Bool {
 }
 
 extension HTTPConnection: Hashable {
-    public var hashValue: Int {
-        return uuid.hashValue
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(uuid.hashValue)
```

Comment:
@keith hmm, I thought `uuid` is also a Hashable, can't we pass `uuid` instead of `uuid.hashValue`?

Reply:
Done!